main.json

```diff
@@ -0,0 +1,39 @@
+[
+{
+    "name":"Yochai Benkler",
+    "role":"Berkman Professor",
+    "details":"Learn modern ways of protecting your system",
+    "image": "./assets/project/yochai benkler.png"
+},
+{
+    "name":"Julia Leda",
+    "role":"Berkman Professor",
+    "details":"Learn modern ways of protecting your system",
+    "image": "./assets/project/julia-leda.png"
+},
+{
+    "name":"Kilman Chon",
+    "role":"Berkman Professor",
+    "details":"Learn modern ways of protecting your system",
+    "image": "./assets/project/Kilman-chon.png"
+},
+{
+    "name":"Lila Trekikov",
+    "role":"Berkman Professor",
+    "details":"Learn modern ways of protecting your system",
+    "image": "./assets/project/lila-tretikov.png"
+},
+{
+    "name":"Ryan Merkley",
+    "role":"Berkman Professor",
+    "details":"Learn modern ways of protecting your system",
+    "image": "./assets/project/Ryan-merkley.png"
+},
+{
+    "name":"Sohyeong Noh",
+    "role":"Berkman Professor",
+    "details":"Learn modern ways of protecting your system",
+    "image": "./assets/project/sohyeong-Noh.png"
+}
```

Comment:
[OPTIONAL] Your project will look nice and more proffessional if the `role` and `details` values for each speaker is distinct and not repeated